core/src/main/java/nl/nn/adapterframework/scheduler/ConfiguredJob.java

```diff
@@ -55,9 +55,9 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
 			JobDataMap dataMap = context.getJobDetail().getJobDataMap();
 			IJob jobDef = (IJob)dataMap.get(JOBDEF_KEY);
 			Thread.currentThread().setName(jobDef.getName() + "["+ctName+"]");
-			log.info(getLogPrefix(jobDef) + "executing");
+			log.trace(getLogPrefix(jobDef) + "executing");
 			jobDef.executeJob();
-			log.debug(getLogPrefix(jobDef) + "completed");
+			log.trace(getLogPrefix(jobDef) + "completed");
```

Comment:
Heel eigenlijk (zeker als dit nu trace is) zou hier of een isTraceEnabled voor moeten.